### PR TITLE
Add a repo list

### PR DESCRIPTION
**Pull request by daniel-hutao**
Signed-off-by: daniel-hutao <farmer.hutao@outlook.com>

I have completed a preliminary version here, welcome everyone to improve it together.

#318

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
I'd prefer to have a separate file rather than put this table in a readme file.

**Comment by daniel-hutao** [comment]
> Thanks for taking the initiative. I'd prefer to have a separate file rather than put this table in a readme file.

It is appropriate to maintain these contents on a separate page, and then maintain a link in the README.md file; in addition, we need to place a list of core code repos on the official website. These things are a bit tiring for me. It is recommended to specify several internal developments who relatively more familiar with the existing code repos to do this.

**Comment by LinuxSuRen** [comment]
No worries. We can take our time to do this.

File: README.md [hunk 1]
```diff
@@ -22,3 +22,43 @@ Working Group (WG) facilitates discussions and work on cross-SIGs. For example,
 - [Twitter](https://twitter.com/KubeSphere)
 - [Mailing List](https://groups.google.com/forum/#!forum/kubesphere)
 - [Medium (Blog)](https://itnext.io/@kubesphere)
+
+## Repository list // WIP
```
Comment (LinuxSuRen) [inline README.md:26]: > Repository list // WIP

I'd like to change it to `Repository list (WIP)`

File: README.md [hunk 2]
```diff
@@ -22,3 +22,43 @@ Working Group (WG) facilitates discussions and work on cross-SIGs. For example,
...
+
+|      |      |
+| ---- | ---- |
+| Project | Description |
```
Comment (LinuxSuRen) [inline README.md:30]: This line should be in line 28. And line 28 should be here.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for providing the organized projects list. But I'd suggest we keep this README file clean. For example, move the projects list into a separate file.

File: README.md (hunk 2 above)
Comment (daniel-hutao) [reply README.md:30]: As you said earlier, we hope to keep the README file clean, and we have discussed that QingCloud can assign one member who familiar with the projects list in ks org to organize the first version of the repo list, so I don’t want to summit a new commit for this issue.

Comment (LinuxSuRen) [reply README.md:30]: In my opinion, we still can merge this PR and push it forward. The most important of community is improvement instead of perfect.

Comment (LinuxSuRen) [reply README.md:30]: I will take over this PR if you're not interested in this.

**Comment by LinuxSuRen** [comment]
Considering this PR has not responded for a long time. I'm going to close it.

If there's anyone wants to get a list of DevOps-related repositories, please check [here](https://github.com/kubesphere/ks-devops/pull/172).

/close

**Review by LinuxSuRen — COMMENTED** [review]
Considering this PR has not responded for a long time. I'm going to close it.

If there's anyone wants to get a list of DevOps-related repositories, please check [here](https://github.com/kubesphere/ks-devops/pull/172).

/close

**Pull request closed without merging**